.github/config/chart-releaser.yaml

```diff
@@ -0,0 +1 @@
+release-notes-file: RELEASE-NOTES.md
```

Comment:
:question: Is it enought to just specify the name of do we need to specify the path `charts/*/RELEASE-NOTES.md` ?

Reply:
since I was the one who introduced this feature in Helm chart-releaser, the answer is yes :D 
it looks for files within the chart dir.